### DISCUSSION TITLE
Unexpected high content download time

### DISCUSSION THREAD

**Discussion by vicentel89** [vercel/next.js#63211] [App Router]
With a little change in a client component that is inside a server component, the time to update is too high.

![image](https://github.com/vercel/next.js/assets/60248266/0ec782a0-1294-40d7-be88-2c3ddc3ec15f)

It is strange that the size for the ´ localhost´ file size is only 8kb but it takes a lot of time more than the others that are bigger.

Another thing that is unexpected is that if I make the little change again, it downloads the same css files again.
![image](https://github.com/vercel/next.js/assets/60248266/6794f064-5c18-4c61-b9fd-dfc97c2a1582)

Version 14.1.3
App Router

**Comment by vicentel89** [comment]
https://github.com/vercel/next.js/assets/60248266/5bc32cb8-b26d-4206-98da-6868f012dc90

The update is triggered here https://github.com/vicentel89/marvel-characters/blob/main/src/app/_components/favorite-button/index.tsx

**Comment by NiazMorshed2007** [comment]
Facing the same.

**Comment by farid-ouachrar** [comment]
Same here 

<img width="1165" alt="image" src="https://github.com/user-attachments/assets/81656f12-960c-47b0-a031-e5c124251911">

**Comment by michaelgrewal** [comment]
I'm facing this same issue but only on my Production and not in my Stage.

**Comment by valentindiehl** [comment]
Same issue here on production

**Comment by Ivor808** [comment]
yep same issue
